src/main/webapp/scholarx.html

```diff
@@ -31,11 +31,11 @@
 <section class="section pb-0 bg-gradient-white">
     <div class="container">
         <div class="row row-grid align-items-center justify-content-center">
-            <div class="col-md-8 order-lg-1 text-center">
+            <div class="col-md-6">
                 <div class="d-flex px-3">
                     <div class="pl-4">
                         <img src="assets/img/brand/scholarx.png" class="img-fluid align-self-center mb-5"
-                             style="width: 40%">
+                             style="width: 50%">
```

Comment:
@BhathiyaTK 
It's a great great job you have done with the output. However, as a best practice we try to use less inline CSS codes.
Thus it'll be awesome and more readable if we can use bootstrap classes.
Try using w-50 class here.
- Read this documentation for a better understanding https://getbootstrap.com/docs/4.4/utilities/sizing/#relative-to-the-parent

Cheers Mate!
For betterment!!

Reply:
😍 Sure. I will update.